chapters.py

```diff
@@ -9,222 +9,255 @@
 
 colorama.init(convert=True)
 
+
 # start the game
 
 
 def start():
     answer = input(
         Fore.GREEN +
-        "You are on a dirt road. Which way to you want to go left or right? " +
+        "You are on a dirt road \U0001F6E3. Which way do you want to go: left/l \U000021AA or right/r \U000021A9? : " +
         Fore.LIGHTMAGENTA_EX).lower()
     # if user inputs left then
-    if answer == "left":
-        random.choice(my_list)()
+    if answer == "left" or answer == "l":
+        chapter_lake()
 
     # if user inputs right then
-    if answer == "right":
-        random.choice(my_list)()
+    if answer == "right" or answer == "r":
+        chapter_mountain()
 
 
 def chapter_river():
     answer = input(
         Fore.GREEN +
-        "You come to a river, you can walk around it or swim across."
-        "Type walk to walk around & swim to swim across. " +
+        "\nYou have come across a river. You can either walk around it or swim across it.\n"
+        "Type \"walk/w\" to walk around or \"swim/s\" to swim across. " +
         Fore.LIGHTMAGENTA_EX).lower()
     # if user inputs swim then
-    if answer == "swim":
+    if answer == "swim" or answer == "s":
         game_over(
             Fore.RED +
-            "You swam across the river and were eaten by an aligator \U0001F480"
+            "You swam across the river and were eaten by an alligator \U0001F480"
         )
     # if user inputs walk then
-    elif answer == "walk":
+    elif answer == "walk" or answer == "w":
         # q2
         answer = input(
             Fore.GREEN +
-            "You walked for many miles, ran out of water and remembered "
-            "that there was a shop far away (10 miles/16kms) which supplies water."
-            "Do you want to go there (yes/no)? " +
+            "You have walked many miles and run out of water. "
+            "Fortunately, there is a shop located 10 miles (16kms) away that supplies water. "
+            "Would you like to go there? (yes/no): " +
```

Comment:
I would prefer if it would be (y/n) in the bracket but also accept yes/no, it makes it easier for the user. Otherwise, even if it accepts y/n, the user may not know and will be forced to type the entire thing, which some people may not want to do.

Reply:
yes but can you make it in the bracket (y/n) and then if user enters yes or no it will work?